server.js

```diff
@@ -41,3 +45,34 @@ io.on('connection', function(client){
     ClientManagerService.disconnected(client);
   });
 });
+function getGameServerPath(){
+    if (fs.existsSync('./config/config.json')) {
+        console.log("Checking your config file");   
+        var config;
+        fs.readFile('./config/config.json', function read(err, data) {
+          if (err) {
```

Comment:
I'd be fine with 4 space indentation across all the files. If you choose to do that, please do that in a separate standalone PR